index.html

```diff
@@ -17,5 +18,68 @@
 
     <!-- All the imges you need are in img folder -->
 
+    <nav class="header col-12">
+        <div class="col-8">
+            <ul>
+                <li>Login</li>
```

Comment:
Your `ul` and `li` items could do with some grid classes to help with the layout.